frontend/src/components/Graph/utils.ts

```diff
@@ -97,7 +99,7 @@ export const getGraphOptions = (
 			callbacks: {
 				title(context): string | string[] {
 					const date = dayjs(context[0].parsed.x);
-					return date.format('MMM DD, YYYY, HH:mm:ss');
+					return date.tz(timezone?.value).format('MMM DD, YYYY, HH:mm:ss');
```

Comment:
What will be return for date.tz(undefined) ?

Reply:
I removed the optional chaining, the timezone.value is always going to have a value (either a selected timezone or browser timezone)